GUI/Types/Renderer/Shaders/simple.frag

```diff
@@ -77,10 +76,6 @@ vec3 calculateWorldNormal()
     vec3 tangentNormal = oct_to_float32x3(temp);
 #endif
 
-#if param_LegacySource1InvertNormals == 1
-    tangentNormal.y *= -1.0;
```

Comment:
Are we sure about this in HLA/SteamVR and stuff?

Reply:
The HLA door normal mentioned in #525 does appear to be in OpenGL format after removing invert on Decode (looking at the image). It matches with other normalmaps that don't have Source1 Invert. Since the VRF renderer is OpenGL it works nicely. S&box devs have mentioned that all their DirectX shaders have to do Y inversion.

Since inverted Y normals is an artist choice I don't think they have any logic in their shaders besides the having to flip on Vulkan.

Reply:
@lewa-j I doubt you remember, but do you know why we had to invert normals in the first place?

It was added in https://github.com/SteamDatabase/ValveResourceFormat/commit/10bf93bcf4fe39b761341484fbc37e6347516341 but it mentions hemioct, which is separate.